control/tests/sisotool_test.py

```diff
@@ -46,6 +46,8 @@ def sys221(self):
         D221 = [[1., -1.]]
         return StateSpace(A222, B222, C221, D221)
 
+    @pytest.mark.skipif(plt.get_current_fig_manager().toolbar is None,
+                        reason="Requires the zoom toolbar")
```

Comment:
It looks like this skip never actually gets triggered in the runs.  Seems fine to leave, but just wanted to confirm that was expected.

Reply:
It gets triggered in the Slycot from src job where there is no explicit xvfb-run or pytest-xvfb installed. The default on the headless GHA runner is `Agg` there, whereas inside the conda pytest-xvfb it seems to be already an interactive backend (QtAgg) by default.